File: js/task2/S2_FOR_04.js
// S2_FOR_04

function countOccurrences(values) {
    let counts = {};
    
    for (let v of values) {
        if (counts[v]) {
            counts[v]++;
        } else {
            counts[v] = 1;
        }
    }
    return counts;
}

console.log(countOccurrences(["apple", "banana", "apple"]));

File: python/task2b/S2_FOR_04.py
# S2_FOR_04

def count_occurrences(values):
    counts = {}
    for v in values:
        if v in counts:
            counts[v] += 1
        else:
            counts[v] = 1
    return counts

print(count_occurrences(["apple", "banana", "apple"]))